csaf_2.0/submit/README.md

```diff
@@ -0,0 +1,84 @@
+   *stages of approval and who votes at each stage.*
+
+Intended status: Advance CSAF\ v2.0 OASIS to an ISO/IEC International Standard
+
+Process: ISO/IEC\ JTC\ 1 PAS Transposition Process involving a Draft International
+Standard\(DIS) ballot followed by a Ballot Resolution Meeting\ (BRM) (if there are comments
+during ballots), and potentially a Final DIS\ (FDIS) Ballot.
+
+Stages of approval: https://www.iso.org/stage-codes.html - Stage 40
+
+
+*Note: Submission request continued on next page*
+
+                                              *... continued from first page of submission request.*
+
+d. *An explanation of how the submission will benefit OASIS.*
+
+Advancing CSAF\ v2.0 to an international standard draws international attention to the work
+done by OASIS and to the organization itself,and governments/regulators may be more
+inclined to use the standard because of its ISO/IEC status.
+
+This will help nurture and expand the ongoing liaisons with ISO/IEC JTC 1.
+
+Further, this submission will help maintain OASIS in good standing as a recognized ISO/IEC
+JTC\ 1 PAS submitter organization.
+
+Last, but not least, as ISO/IEC standard, the process requirements for the success of the
+CSAF\ v2.0 OASIS Standard will be more easily enforceable through certification and audit.
+
+Security advisories still require essential inputs from human experts within organizations and
+rely on organizational processes that plausibly implement the preconditions for successful
+implementation.
+
+e. *The expected licensing, copyright and other intellectual property terms that will be used by the receiving*
+   *organization in regard to the submission.*
+
+ISO/IEC JTC\ 1 will expect that OASIS will abide by the ISO/IEC/ITU-T Common Patent Policy
+(indicated by at least notifying ISO/IEC of any patent declarations in the required Explanatory
+Report), and will provide ISO/IEC with sufficient copyright license to modify and publish the
+resulting ISO/IEC Standard.
+
+As these items have already been agreed with ISO/IEC JTC\ 1 when OASIS was approved as a
+PAS Submitter, no new issues are expected.
+
+f. *A statement of the intended future plans for versioning and maintenance of the OASIS Standard and/or*
+   *Approved Errata for that standard, and the expected roles of OASIS and the receiving organization. This*
+   *must include clear statements of the rules of the receiving organization applicable to maintenance of an*
+   *approved submitted standard, and to future versions of that standard; any requirements regarding the*
+   *submission of future versions; and a description of how OASIS and the submission requesters expect to*
+   *comply with those rules.*
+
+The OASIS OData Technical Committee will continue to maintain the CSAF\ v2.0 OASIS
+Standards and produce future revisions of them.
+
+For this specific submission (CSAF\ v2.0 OASIS) to ISO/IEC JTC\ 1, OASIS will request to be
```

Comment:
We are not submitting OASIS but an OASIS Standard &rightarrow; fixed in [570fc62a...](https://github.com/oasis-tcs/csaf/pull/603/commits/570fc62a312941f5fdc9706f1bcfab549b5dea37).